src/OnboardingSPA/components/Drawer/DrawerPanel/DesignTypography.js

```diff
@@ -75,20 +56,20 @@ const DesignTypography = () => {
 
 	const handleClick = async (
 		fontStyle,
-		selectedGlobalStyle = globalStyles,
+		selectedGlobalStyle = storedPreviewSettings,
 		fontPalettesCopy = fontPalettes
 	) => {
 		setSelectedFont( fontStyle );
 
 		// Changes the Global Styles to Recompute css properties
 		const globalStylesCopy = selectedGlobalStyle;
-		globalStylesCopy.styles.typography.fontFamily =
-			fontPalettesCopy[ fontStyle ]?.styles?.typography?.fontFamily;
-		globalStylesCopy.styles.blocks[ 'core/heading' ].typography.fontFamily =
-			fontPalettesCopy[ fontStyle ]?.styles.blocks[
-				'core/heading'
-			].typography.fontFamily;
-		setGlobalStyles( globalStylesCopy );
+
+		// globalStylesCopy.styles.typography.fontFamily =
+		// 	fontPalettesCopy[ fontStyle ]?.styles?.typography?.fontFamily;
+		// globalStylesCopy.styles.blocks[ 'core/heading' ].typography.fontFamily =
+		// 	fontPalettesCopy[ fontStyle ]?.styles.blocks[
+		// 		'core/heading'
+		// 	].typography.fontFamily;
```

Comment:
We can remove this code?

Reply:
So this is the part which will work once the use-global-styles is fixed to work like before, I can sure go ahead and remove this if you want.